### ISSUE TITLE
devpod wrongly shows informational messages from dockerless build as errors

### ISSUE THREAD

**Issue by mgolisch** [loft-sh/devpod#818]
**What happened?**  
devpod up output shows alot of errors when building the image when there actualy is no errors at all

**What did you expect to happen instead?**  
It should only show errors if there is actualy an error

**How can we reproduce the bug?** (as minimally and precisely as possible)  
create a workspace with a devcontainer.json that uses a dockerfile with the kubernetes provider
My *`devcontainer.json`*:
```
{
    "name": "asdf",
    "build": {
        "dockerfile": "Dockerfile"
    }
}
```

**Local Environment:**  
- DevPod Version: v0.4.2
- Operating System: windows
- ARCH of the OS: AMD64

**DevPod Provider:**  
- Kubernetes Provider:  kubectl: v1.27.7
- Probably any other provider that uses dockerless

**Anything else we need to know?**

**Comment by pascalbreuninger** [comment]
@mgolisch thanks for reporting, we'll change that

**Comment by FabianKramm** [comment]
@mgolisch this should be fixed with the latest devpod alpha build (v0.5.0-alpha.0)
